gitpractice/javascript/task1.js

```diff
@@ -0,0 +1,4 @@
+/*http://www.codewars.com/kata/printing-array-elements-with-comma-delimiters*/
+function printArray(array){
+    return array.join(",");
```

Comment:
Array method join, by default take comma(,) as the separator.
No need to specify a separator if you want to separate them by comma.

array.join();

Look into the document link below to learn more about array.join():
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

They is nothing wrong in specifying the separator comma.

Reply:
yeah i modiefied it.